### PR TITLE
Legalization and Drug Deaths

### PR DESCRIPTION
**Pull request by tmarchBU**

### CODE REVIEW

**Review by gallettilance — CHANGES_REQUESTED** [review]
please nest your proposal within a folder named after your team name instead of having the file in the root of the repository

**Comment by tmarchBU** [comment]
Should be fixed now, if this isn't the configuration you're looking for, please let me know

**Review by gallettilance — CHANGES_REQUESTED** [review]
structure looks good. The proposal statement needs a bit of work. For now can you rephrase the problem statement to something along the lines of "looking at the impact of legalization on:
1. drug related deaths
2. economic factors you have data for / are interested in
3. other factors you are interested in"
what do you think?

**Comment by tmarchBU** [comment]
Is this specifically about the Project Description section? If so, I agree, it could definitely do with some re-wording. Something along the lines of:
"Our project seeks to determine the effect of cannabis/marijuana legalization (both medicinal and recreational) on drug-related deaths. Do we see an increased rate due to increased drug use, or do we see a decreased rate due to cannabis products being used as substitutes to other drugs, more resources for people with drug-related issues, and other effects of cannabis legalization."

Is this what you mean?

**Comment by gallettilance** [comment]
@tmarchBU yes

**Review by gallettilance — CHANGES_REQUESTED** [review]
please change the wording

**Review by gallettilance — APPROVED** [review]
Great start but beyond finding a mathematic correlation between these two variables, please look at this from a more socio-economic perspective to understand what exactly is the impact of legalization and can its impact on death be measures indirectly through other variables?
